tests/azure/go.mod

```diff
@@ -0,0 +1,106 @@
+module github.com/fluxcd/flux2/tests/azure
+
+go 1.17
```

Comment:
Please use go 1.16 bumping the version to 1.17 should happen first in fluxcd/pkg then controllers and lastly flux2.